app/views/addCustPaymentOptV.js

```diff
@@ -3,7 +3,7 @@ module.exports.paymentOptionPrompts = [
     name: 'paymentType',
     description: 'Please enter a payment option',
     required: true,
-    type: 'string'
+    pattern: '^[0-9 ]*$'
```

Comment:
So this doesn't entirely restrict user to inputting to available payment type ids. We currently have payment types up to `18`. This regex would still allow a user to input `21` or `3214129` or whatever. We could throw in a `conform` here instead to make sure the payment type is restricted to the available ids.

Reply:
#70